### PR TITLE
Add fallback command for generating support bundle with replicated

### PR DESCRIPTION
**Pull request by wissem**

### CODE REVIEW

File: docs/enterprise/troubleshooting.md [hunk 1]
```diff
@@ -14,7 +14,18 @@ the logs in this file yourself as well.
 You can generate a support bundle from the Replicated Management Console on
 port 8800 by navigating to the Support tab and clicking on the "Download
 Support Bundle" button. Alternatively you can use replicated's command line
-interface to generate the support bundle on your host system: `replicatedctl support-bundle`
+interface to generate the support bundle on your host system: `replicatedctl support-bundle` and if that doesn't work, you can run the following command:
```
Comment (glaubinix) [inline docs/enterprise/troubleshooting.md:17]: I wonder whether we should mention that the command generates a minimal support bundle as it doesn't contain all the information from the regular support bundle (to make sure people not just keep using this one) and add the command will ask them if they want to upload the support bundle. If they do they still need to let us know as we do not get any notification. If they decide not to upload they can send it as normal.

Comment (wissem) [reply docs/enterprise/troubleshooting.md]: yes it makes sense to mention that the fallback generates a minimal support bundle

**Review by glaubinix — APPROVED** [review]

File: docs/enterprise/troubleshooting.md (hunk 1 above)
Comment (naderman) [reply docs/enterprise/troubleshooting.md]: Can you please explain what minimal means in the way you described it here? Someone reading the docs will likely not understand what "minimal" refers to. It should explain that it contains less information about the application, but can be helpful to debug a problem if the full bundle cannot be generated.

Comment (naderman) [reply docs/enterprise/troubleshooting.md]: Further please actually explain the command will interactively ask to upload the bundle but that they have to let us know, as Stephan requested?

Comment (wissem) [reply docs/enterprise/troubleshooting.md]: sorry I missed that. Added here https://github.com/packagist/private-packagist-docs/pull/39

**Pull request merged**